src/home.js

```diff
@@ -0,0 +1,159 @@
+export function renderHeader() {
+  const contentID = document.getElementById('content');
+  contentID.innerHTML += `  
+                <div class="header">
+                <ul class="header-tabs">
+                <li class="logo">
+                    <a id="logo" href="#home"> Zacks </a>
+                </li>
+                <li class="list-item" id="home-tab">Home</li>
+                <li class="list-item" id="menu-tab">Menu</li>
+                </ul>
+            </div>
```

Comment:
Awesome Design 👍 
Seems like you missed the hard requirement for this project which is making use of `createElement` and `appendChild`.


checkout the screenshot from Odin page below;
![image](https://user-images.githubusercontent.com/31897434/80676484-c444b080-8aae-11ea-8245-1ab7c09d3c14.png)
**create them by using JavaScript only, e.g. by appending each new element to div#content once the page is first loaded**

Kindly Use `document.createElement` and `document.appendChild`